src/data/nat/lattice.lean

```diff
@@ -141,6 +141,9 @@ begin
   { exact hb }
 end
 
+lemma Inf_le_Inf_iff_subset {P Q : set ℕ} (h₁ : Q.nonempty) (h₂ : Q ⊆ P) : Inf P ≤ Inf Q :=
```

Comment:
This lemma name includes `iff`, but the statement does not.

Reply:
Fixed